test/core/end2end/fuzzers/api_fuzzer.cc

```diff
@@ -118,7 +128,7 @@ void my_resolve_address(const char* addr, const char* /*default_port*/,
   r->on_done = on_done;
   r->addrs = addrs;
   grpc_timer_init(
-      &r->timer, GPR_MS_PER_SEC + grpc_core::ExecCtx::Get()->Now(),
+      &r->timer, MAX_ADVANCE_TIME_MICROS/1000 + grpc_core::ExecCtx::Get()->Now(),
```

Comment:
MS_PER_SEC is probably better here... it's some arbitrary hardcoded constant that we should replace with a fuzzer selected value at some point

Reply:
Restored.